lib/active_record/connection_adapters/postgis/schema_statements.rb

```diff
@@ -5,18 +5,17 @@ module ConnectionAdapters
     module PostGIS
       module SchemaStatements
         # override
-        # https://github.com/rails/rails/blob/6-0-stable/activerecord/lib/active_record/connection_adapters/postgresql/schema_statements.rb#L624
+        # https://github.com/rails/rails/blob/7-0-stable/activerecord/lib/active_record/connection_adapters/postgresql/schema_statements.rb#L662
         # Create a SpatialColumn instead of a PostgreSQL::Column
         def new_column_from_field(table_name, field)
-          column_name, type, default, notnull, oid, fmod, collation, comment = field
+          column_name, type, default, notnull, oid, fmod, collation, comment, attgenerated = field
           type_metadata = fetch_type_metadata(column_name, type, oid.to_i, fmod.to_i)
           default_value = extract_value_from_default(default)
           default_function = extract_default_function(default_value, default)
 
-          serial =
-            if (match = default_function&.match(/\Anextval\('"?(?<sequence_name>.+_(?<suffix>seq\d*))"?'::regclass\)\z/))
-              sequence_name_from_parts(table_name, column_name, match[:suffix]) == match[:sequence_name]
-            end
+          if match = default_function&.match(/\Anextval\('"?(?<sequence_name>.+_(?<suffix>seq\d*))"?'::regclass\)\z/)
+            serial = sequence_name_from_parts(table_name, column_name, match[:suffix]) == match[:sequence_name]
+          end
```

Comment:
I'd rather not include cosmetic changes in meaningful (eg not fully cosmetic) PRs, this only make reviewing and blaming harder unfortunately, plus that is opiniated (for instance, I favor parenthis in if conditions with assignment to distinguish those).. Could you revert that bit please?

Reply:
This is a verbatim copy from: https://github.com/rails/rails/blob/ba8d029e0c9965f64c96f44326fe40f0df6293f3/activerecord/lib/active_record/connection_adapters/postgresql/schema_statements.rb#L668-L670

The change is cosmetic but I think it is important to keep this method as close as possible to the Rails codebase since we're overwriting it.  Should I revert it still?

Reply:
Oh yeah makes definitely more sense then to let it asis, thanks for the explanation :)